place_tracker/lib/place_details.dart

```diff
@@ -140,10 +145,13 @@ class _NameTextField extends StatelessWidget {
 }
 
 class _DescriptionTextField extends StatelessWidget {
-  _DescriptionTextField({
+  const _DescriptionTextField({
+    Key key,
```

Comment:
nit: I generally like to put the required ones first, then the others (particularly key, which is just getting passed through).

Reply:
Done.